pom.xml

```diff
@@ -305,6 +308,15 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.sling</groupId>
+                <artifactId>maven-sling-plugin</artifactId>
+                <version>2.1.0</version>
+                <configuration>
+                    <slingUrl>http://${aem.host}:${aem.port}${aem.contextPath}/system/console</slingUrl>
```

Comment:
Why don't we use the `sling.url` attribute directly? Do we ever want to use the host / port / context path as separate attributes?